### ISSUE TITLE
Refresh Train Depot HUD After Editing Train

### ISSUE THREAD

**Issue by Phelzier** [voxeltycoon/bug-tracker#76]
I have a train with two locomotives facing the same way.  After editing the train and flipping the end locomotive to face the opposite way, the HUD did not reflect these changes.  Closing the HUD and reopening showed the locomotive in the correct direction.

![bug](https://user-images.githubusercontent.com/22050004/60390012-eb180c00-9ac4-11e9-91ed-c2cef006ea6e.png)

**Comment by polak-jan** [comment]
This is probably the same bug as #24, I think it would be better to move this to a comment on that one and close this one.

Also please use the bug report template in your future bug reports.

**Comment by Phelzier** [comment]
added to #24.  Also description seems similar to closed issue #34

**Comment by andrewpey** [comment]
@Phelzier @polak-jan yeah, it's more close to #34, however that was fixed, so it's probably a new bug, reopening.
